website/screens/principles/iconography/usage/IconographyUsagePage.tsx

```diff
@@ -0,0 +1,97 @@
+import DocFooter from "@/common/DocFooter";
+import QuickNavContainer from "@/common/QuickNavContainer";
+import QuickNavContainerLayout from "@/common/QuickNavContainerLayout";
+import { DxcBulletedList, DxcFlex } from "@dxc-technology/halstack-react";
+import Image from "@/common/Image";
+import icons from "./images/Iconography_image.png";
+import Link from "next/link";
+
+const sections = [
+  {
+    title: "Usage",
+    content: (
+      <>
+        <Image src={icons} height={400} alt="Halstack Icons" />
+        <DxcBulletedList>
+          <DxcBulletedList.Item>
+            Icon sizes can vary based on use case and application context. Most
+            of the icons that are currently used within Halstack components
+            follow a 24x24px size including the background or space around the
+            icon. There are a few exceptions where 20x20 or 16x16 icon sizes are
+            used as these sizes better match the visual context of the element.
+          </DxcBulletedList.Item>
+
+          <DxcBulletedList.Item>
+            Currently, the default color of icons is always inherited from the
+            parent during implemented. If no color is specified, the icon will
+            black by default (with a few cases where the default is purple).
+          </DxcBulletedList.Item>
+          <DxcBulletedList.Item>
+            Icons come in outlined and filled variants. The currently
+            implemented icons use base colors of black or white for visual
+            consistency.
+          </DxcBulletedList.Item>
+          <DxcBulletedList.Item>
+            While there currently exists a library of over three thousand icons,
+            there may be instances when a custom icon is needed (such as icons
+            that represent brands, companies, or organizations), an SVG
```

Comment:
remove comma next to companies

Reply:
with or without comma before the end - both approaches are acceptable:

https://www.ego4u.com/en/cram-up/writing/comma?08#:~:text=Use%20a%20comma%20to%20separate,before%20%E2%80%9Cand%E2%80%9D%20is%20optional.